allium/static/css/allium-base.css

```
@@ -0,0 +1,613 @@
+/* Create tooltip content using title attribute */
+[title]::before {
+    content: attr(title);
+    position: absolute;
+    background: rgba(0, 0, 0, 0.9);
+    color: white;
+    padding: 8px 12px;
+    border-radius: 6px;
+    font-size: 13px;
+    line-height: 1.3;
+    max-width: 300px;
+    word-wrap: break-word;
```

Stylelint (17.4.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)